test/distrib/csharp/DistribTest/DistribTest.csproj

```diff
@@ -100,7 +100,7 @@
     <Reference Include="Google.Apis.Auth.PlatformServices">
       <HintPath>..\packages\Google.Apis.Auth.1.15.0\lib\net45\Google.Apis.Auth.PlatformServices.dll</HintPath>
     </Reference>
-    <Reference Include="Google.Protobuf, Version=3.12.2, Culture=neutral, PublicKeyToken=a7d26565bac4d604, processorArchitecture=MSIL">
+    <Reference Include="Google.Protobuf">
```

Comment:
other assemblies also don't explicitly mention their version. HintPath (and packages.config file) should be enough